src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,50 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_LENGTH_STRING = 6;
+    private static final int MIN_USER_AGE = 18;
+    private static final int MAX_USER_AGE = 200;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null) {
+            throw new RegistrationException("User must not be null!!!");
```

Comment:
same for other messages
```suggestion
            throw new RegistrationException("User must not be null");
```